Gopkg.toml

```diff
@@ -27,37 +27,36 @@ required = [
   branch = "master"
 
 [[override]]
-  name = "k8s.io/kube-aggregator"
-  branch = "release-1.10"
+  name = "github.com/openshift/api"
+  branch = "master"
 
 [[override]]
   name = "sigs.k8s.io/cluster-api"
   branch = "master"
 
+[[override]]
+  name = "k8s.io/kube-aggregator"
+  version = "kubernetes-1.11.1"
+
 [[override]]
   name = "k8s.io/code-generator"
-  # revision for tag "kubernetes-1.11.2"
-  revision = "6702109cc68eb6fe6350b83e14407c8d7309fd1a"
+  version = "kubernetes-1.11.1"
```

Comment:
Why not `1.11.2`?

Reply:
I checked other openshift repositories that we depend on like `github.com/openshift/api`, `github.com/openshift/client-go` and they use `kubernetes-1.11.1`

Reply:
fair enough. Hopefully there is no signification difference (e.g. known errors in 1.11.1 fixed in 1.11.2).

Reply:
@ingvagabund, in the end, I updated dependencies to 1.11.2 because `controller-runtime` depends on it